native_client/javascript/Makefile

```diff
@@ -1,6 +1,7 @@
 NODE_BUILD_TOOL ?= node-pre-gyp
 NODE_ABI_TARGET ?= 
 NODE_BUILD_VERBOSE ?= --verbose
+TFDIR ?= ../../../../tensorflow
```

Comment:
We already have that variable defined in `definitions.mk`, maybe we should change it there with some expansion to an absolute path ?